src/gm.datepickerMultiSelect.js

```diff
@@ -48,7 +48,7 @@ SOFTWARE.
 
 					/* Fires when date is selected or when month is changed. */
 					scope.$watch(function () {
-						return ctrl.activeDate.getTime();
+						return ctrl.activeDate ? ctrl.activeDate.getTime() : false;
```

Comment:
Why this change? You don't want to watch 'false,' and it will silently fail if ctrl.activeDate isn't defined.